plugins/CoreHome/javascripts/sparkline.js

```diff
@@ -69,7 +70,11 @@ window.initializeSparklines = function () {
                     // if this happens, we can't find the graph using $('#'+idDataTable+"Chart");
                     // instead, we just use the first evolution graph we can find.
                     if (dataTable.length == 0) {
-                        dataTable = $('div.dataTableVizEvolution');
+                        if ($(this).parents('.widget').length) {
```

Comment:
Would `.closest(` be better here instead of `.parents(`? I assume we only need the first `.widget` parent.